packages/api/test/analytics-test.js

```diff
@@ -94,44 +94,10 @@ describe('Flaky-Analytics', () => {
     it('Should track stats for tests', async () => {
       const testFlakyExpecations = [false, true, true, false];
       for (let i = 0; i < testFlakyExpecations.length; i++) {
-        const testData = await client.collection(global.headCollection).doc(buildInfo.repoId).collection('tests').doc(i.toString()).get();
+        const testData = await client.collection(global.headCollection).doc(buildInfo.repoId).collection('queued').doc(i.toString()).get();
         assert.strictEqual(testData.data().flaky, testFlakyExpecations[i]);
       }
     });
-
```

Comment:
this test was removed considering we are assuming builds will be uploaded chronologically